src/devices/Bmxx80/samples/Bme280.sample.cs

```diff
@@ -69,13 +77,20 @@ static void Main(string[] args)
 
                     //read values
                     i2CBmpe80.TryReadTemperature(out tempValue);
-                    Console.WriteLine($"Temperature: {tempValue.Celsius} \u00B0C");
                     i2CBmpe80.TryReadPressure(out preValue);
-                    Console.WriteLine($"Pressure: {preValue.Hectopascal} hPa");
-                    i2CBmpe80.TryReadAltitude(defaultSeaLevelPressure, out altValue);
-                    Console.WriteLine($"Altitude: {altValue} meters");
                     i2CBmpe80.TryReadHumidity(out humValue);
-                    Console.WriteLine($"Humidity: {humValue} %");
+                    altValue = WeatherHelper.CalculateAltitude(preValue, defaultSeaLevelPressure, tempValue);
+                    
+                    Console.WriteLine($"Temperature: {tempValue.Celsius:0.#}\u00B0C");                    
+                    Console.WriteLine($"Pressure: {preValue.Hectopascal:0.##}hPa");                    
+                    Console.WriteLine($"Altitude: {altValue:0.##}m");                    
+                    Console.WriteLine($"Relative humidity: {humValue:0.#}%");
+                    Console.WriteLine($"Heat index: {WeatherHelper.CalculateHeatIndex(tempValue, humValue).Celsius} \u00B0C");
+                    Console.WriteLine($"Summer simmer index: {WeatherHelper.CalculateSummerSimmerIndex(tempValue, humValue).Celsius} \u00B0C");
+                    Console.WriteLine($"Saturated vapor pressure: {WeatherHelper.CalculateSaturatedVaporPressure(tempValue).Hectopascal} hPa");
+                    Console.WriteLine($"Actual vapor pressure: {WeatherHelper.CalculateActualVaporPressure(tempValue, humValue).Hectopascal} hPa");
+                    Console.WriteLine($"Dew point: {WeatherHelper.CalculateDewPoint(tempValue, humValue).Celsius} \u00B0C");
```

Comment:
can't tell looking at github diff but if there is a space before unit please remove (same below)

Reply:
Done.